plasmidseeker/1.3/Dockerfile

```diff
@@ -0,0 +1,67 @@
+# FROM defines the base docker image. This command has to come first in the file
+# The 'as' keyword lets you name the folowing stage. We use `app` for the production image
+FROM --platform=linux/amd64 ubuntu:xenial as app
+
+# ARG sets environment variables during the build stage
+ARG PLASMIDSEEKER_VER="1.3"
+
+# LABEL instructions tag the image with metadata that might be important to the user
+# Optional, but highly recommended
+LABEL base.image="ubuntu:xenial"
+LABEL dockerfile.version="1"
+LABEL software="PlasmidSeeker"
+LABEL software.version=$PLASMIDSEEKER_VER
+LABEL description="A k-mer based program for the identification of known plasmids from bacterial whole genome sequencing reads."
+LABEL website="https://github.com/bioinfo-ut/PlasmidSeeker"
+LABEL license="https://github.com/bioinfo-ut/PlasmidSeeker/blob/master/LICENSE"
+LABEL maintainer="John Arnn"
+LABEL maintainer.email="jarnn@utah.gov"
+
+# RUN executes code during the build
+# Install dependencies via apt-get or yum if using a centos or fedora base
+RUN apt-get update -y && apt-get install -y \
+    git \
+    apt-utils \
+    wget \
+    perl \
+    unzip \
+    jq \
+    gzip \
+    curl \
+    libz-dev && \
+    apt-get autoclean && rm -rf /var/lib/apt/lists/*
+
+# Install R
+RUN apt-get update && apt-get -y upgrade && apt-get -y install r-base \
```

Comment:
Sorry I didn't notice this before, but home come r-base and r-base-dev are installed via apt-get separately?

In theory, I would imagine them to be included somewhere in lines 23-32

Reply:
Installs just fine with everything else.